tomlkit/items.py

```diff
@@ -1675,6 +1728,15 @@ def __init__(
         for table in body:
             self.append(table)
 
+    def unwrap(self) -> str:
+        unwrapped = []
+        for t in self._body:
+            if is_tomlkit(t):
```

Comment:
Also, a child of an array must be an item. `isinstance(t, Item)` is enough for this case. But this also works

Reply:
Done